README.md

```diff
@@ -18,7 +18,7 @@ Agencies are encouraged to use the following HTML snippet to participate in the
 
 ```html
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
-<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
+<script async id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
```

Comment:
Looking at what @afeld filed in https://github.com/digital-analytics-program/gov-wide-code/pull/28, it may be worth not using `async` in the recommended snippet until the technical issues he brought up there are addressed.